templates/data/checkconfig.yaml

```diff
@@ -0,0 +1,50 @@
+{{$exclude := list "template-repository"}}
+templates:
+  - from: templates/generic.tmpl
+    render:
+      - to: ../prow/jobs/test-infra/checkconfig.yaml
+        localSets:
+          default:
+            skip_report: "false"
+            max_concurrency: "10"
+            decorate: "true"
+            command: checkconfig
+            image: gcr.io/k8s-prow/checkconfig:v20220305-a9a1fd8e47
+          jobConfig_inrepo:
+            run_if_changed: "(^.prow/.*.yaml|^.prow.yaml)"
+            args:
+              - "--plugin-config=../prow/plugins.yaml"
+              - "--config-path=../prow/config.yaml"
+              - "--prow-yaml-repo-name=$(REPO_OWNER)/$(REPO_NAME)"
+          jobConfig_test-infra:
+            run_if_changed: "^prow/((plugins|config).yaml|jobs/)"
+            args:
+              - "--plugin-config=prow/plugins.yaml"
+              - "--config-path=prow/config.yaml"
+              - "--job-config-path=prow/jobs"
+        jobConfigs:
+          {{/* Iterate over orgs and repos in config.yaml global field. */}}
```

Comment:
Why do you wrap those in go templates? There's no need to wrap it like that if it's just a comment.

Reply:
I've followed documentation from https://pkg.go.dev/text/template which says:
```
{{/* a comment */}}
{{- /* a comment with white space trimmed from preceding and following text */ -}}
	A comment; discarded. May contain newlines.
	Comments do not nest and must start and end at the
	delimiters, as shown here.
```

Reply:
This is correct, that's true. Do you want this comment to be present in the resulting file? I assumed you wanted to have it as part of the documentation in the resulting file.

Reply:
No, I want this comment to be present in source file only to document what template actions are for. The result of this template execution is not written to disk and it live in memory only for the time of rendertemplate execution. The result is used as data passed to another template to finally generate a prowjob definition.